core/src/main/java/com/linecorp/armeria/common/AbstractHttpHeadersBuilder.java

```diff
@@ -39,6 +44,18 @@ final HttpHeadersBase newSetters(HttpHeadersBase parent, boolean shallowCopy) {
         return new HttpHeadersBase(parent, shallowCopy);
     }
 
+    @Nullable
+    public final List<LanguageRange> acceptLanguages() {
```

Comment:
Shouldn't we put this into `DefaultRequestHeadersBuilder` because it's a request header?

Reply:
Moved to `HttpHeadersBase` as per `uri()` and `path()` and made `package`